auxx/src/Command/Rollback.hs

```diff
@@ -57,7 +58,7 @@ rollbackAndDump numToRollback outFile = do
             liftIO $ BSL.writeFile outFile (serialize allTxs)
             logInfo $ sformat ("Dumped "%int%" transactions to "%string)
                       (length allTxs) (outFile)
-            rollbackBlocksUnsafe (BypassSecurityCheck True) blunds
+            rollbackBlocksUnsafe (BypassSecurityCheck True) (ShouldCallBListener False) blunds
```

Comment:
???

Reply:
chto ti hochech, zachem vizivat ego tut?

Reply:
What's the reason not to call it?

Reply:
Imo, the only reason not to call BListener is when we are sure we will apply the same blocks back. But here it's not the case.

Reply:
ok, fixed. no hz zachem blistener should know about rollback in auxx